.sync/rust_config/config.toml

```diff
@@ -0,0 +1,5 @@
+[target.x86_64-unknown-uefi]
+rustflags = [
+    "-C", "link-arg=/base:0x0",
+    "-C", "link-arg=/subsystem:efi_boot_service_driver",

```

Comment:
nit: you can do a single line with `link-args=/base:0x0 /subsystem:efi_boot_service_driver"`. Don't really care, just if we think that adding future ones is simpler with a single line. That being said, as I type this out I'm changing my mind and I think this is useful to have separated out so that the git blame is easier to identify what came when. So I will just resolve this.

Reply:
Yeah, agree with keeping them separate.